proto/lazyledgerapp/tx.proto

```diff
@@ -0,0 +1,84 @@
+syntax = "proto3";
+package lazyledgerapp;
+
+import "gogoproto/gogo.proto";
+import "google/api/annotations.proto";
+
+option go_package = "github.com/lazyledger/lazyledger-app/x/lazyledgerapp/types";
+
+
+// Msg defines the bank Msg service.
+service Msg {
+    // PayForMessage allows the user to post data to made be available.
+    rpc PayForMessage(MsgWirePayForMessage) returns (MsgWirePayForMessageResponse) {
+        option (google.api.http).get = "/layzyledger/lazyledgerapp/payformessage";
+    }
+  }
+
+// WirePayForMessage describes the format of data that is sent over the wire for
+// each PayForMessage
+message MsgWirePayForMessage {
+  TransactionFee fee = 1;
+  uint64 nonce = 2;
+  bytes message_name_space_id = 3; // assume this is 8 bytes!
+  uint64 message_size = 4;
+  bytes message = 5;
+  repeated ShareCommitAndSignature message_share_commitment = 6 [(gogoproto.nullable) = false];
+  bytes from = 7; // should this be included?
```

Comment:
pinging @adlerjohn regarding `// should this be included?`

I'm wondering if we shouldn't embed a default sdk-tx and only add the LL-specific fields here. Didn't think this through yet though.

Reply:
I don't think you need an explicit `from`. We use ECDSA so you can recover the sender from the signature and whatever's signed over.

Reply:
I've discussed with Evan to keep the from field for now. The rationale for this:
> you need the v value to recover the sender's pubkey as in ethereum/ll-specs: (in one shot): https://github.com/lazyledger/lazyledger-specs/blob/master/specs/data_structures.md#signature (otherwise you will have to try two pubkeys). In tendermint currently only the r||s vals are returned as a sig: https://github.com/tendermint/tendermint/blob/aaf8987257d3fdfda0689b9328f193844ca73d8c/crypto/secp256k1/secp256k1_nocgo.go#L21

We can later change this in ll-core and remove the from field.